### PR TITLE
Refactored Bookmark::get() for DatabaseName

### PR DESCRIPTION
**Pull request by sivadasrajan**
Signed-off-by: SIVADAS <sivadasrajan@gmail.com>

### Description

Refactored Bookmark::get() for DatabaseName and all of its occurances/  as a contibution to #17727

### CODE REVIEW

File: libraries/classes/Sql.php [hunk 1]
```diff
@@ -4,41 +4,42 @@
...
+use PhpMyAdmin\SqlParser\Utils\Query;
+use PhpMyAdmin\ConfigStorage\Relation;
+use PhpMyAdmin\Html\MySQLDocumentation;
+use PhpMyAdmin\ConfigStorage\RelationCleanup;
+use PhpMyAdmin\Display\Results as DisplayResults;
+use PhpMyAdmin\Query\Generator as QueryGenerator;
+use PhpMyAdmin\SqlParser\Statements\DropStatement;
+use PhpMyAdmin\SqlParser\Statements\AlterStatement;
+use PhpMyAdmin\SqlParser\Statements\SelectStatement;
+use PhpMyAdmin\ConfigStorage\Features\BookmarkFeature;
```
Comment (kamil-tekiela) [inline libraries/classes/Sql.php:42]: The `use`s should be ordered by type and alphabetically, not my string length.

Comment (sivadasrajan) [reply libraries/classes/Sql.php:42]: okay. Can you tell me what would be the correct way  to fix this?
Should i close this pull request and submit a new one?

Comment (MauricioFauth) [reply libraries/classes/Sql.php:42]: > okay. Can you tell me what would be the correct way to fix this?

Run the following in your terminal to see coding standard errors:
```
vendor/bin/phps
```
And the following to automatically fix most errors:
```
vendor/bin/phpcbf
```

> Should i close this pull request and submit a new one?

No. You can push new commits into this pull request.

Comment (sivadasrajan) [reply libraries/classes/Sql.php]: Done. Ive updated the PR. I apologize for the delay

File: libraries/classes/Sql.php [hunk 2]
```diff
@@ -481,7 +482,7 @@ public function findRealEndOfRows($db, $table)
      */
     public function getDefaultSqlQueryForBrowse($db, $table): string
     {
-        $bookmark = Bookmark::get($this->dbi, $GLOBALS['cfg']['Server']['user'], $db, $table, 'label', false, true);
+        $bookmark = Bookmark::get($this->dbi, $GLOBALS['cfg']['Server']['user'], DatabaseName::fromValue($db), $table, 'label', false, true);
```
Comment (kamil-tekiela) [inline libraries/classes/Sql.php:485]: Hmm... this should probably be propagated further up, but I am ok with you doing this in the next PR.

**Review by kamil-tekiela — APPROVED** [review]

File: libraries/classes/Sql.php (hunk 2 above)
Comment (sivadasrajan) [reply libraries/classes/Sql.php:485]: > Hmm... this should probably be propagated further up, but I am ok with you doing this in the next PR.

I did not understand. What should be propagated up further? I am new to contributing to opensource and collaborative software development

Comment (kamil-tekiela) [reply libraries/classes/Sql.php:485]: What you did in `Bookmark::get` you can repeat here. Basicelly, the `$db` param doesn't need to be string here. This method can also accept `DatabaseName` as the argument.

Comment (sivadasrajan) [reply libraries/classes/Sql.php:485]: Oh okay.  Now i get the hang of it. I am definitely planning to do more on future pull requests

**Review by MauricioFauth — APPROVED** [review]

**Comment by MauricioFauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**